oc_mirror.sh

```diff
@@ -81,3 +78,74 @@ EOF
    popd
 
 }
+
+function set_full_imageset() {
+
+   # Create imageset with all images from releaseImage
+   imageset=$1
+
+   latest_release=$(oc-mirror list releases --channel candidate-${OPENSHIFT_RELEASE_STREAM}| tail -n1)
+
+   cat > "${imageset}" << EOF
+apiVersion: mirror.openshift.io/v1alpha2
+kind: ImageSetConfiguration
+archiveSize: 16
```

Comment:
I would suggest adding a comment to explain why the archiveSize is set to 16. I think you mentioned otherwise the archive file would be split into multiple files.

Reply:
Done